tests/Integration/NotifierTest.php

```diff
@@ -105,23 +106,49 @@ public function test_sendAlertsPerEmailToRecipient()
 
     public function test_sendAlertsPerEmailToRecipient_shouldUseDifferentSubjectDependingOnPeriod()
     {
-        $this->assertDateInSubject('week', 'week December 21 - 27, 2009');
-        $this->assertDateInSubject('day', 'Thursday, December 31, 2009');
-        $this->assertDateInSubject('month', 'December 2009');
+        $this->assertDateInSubject('week', $this->getMockMail('week')->getSubject());
```

Comment:
>  What I want to achieve is that plugin has green tests for as many Piwik versions as possible or at least 2.15 and 2.16.

it's OK if tests only pass with 2.16.0, as it's likely not possible to get the test to pass on both versions, without removing / skipping this  test on older piwik versions. maybe you leave the test as it is (it's useful to have this integration test here as it can help detect/ prevent a regression), and we only need to make the build green against 2.16.0?